src/reducers/plutus_script_by_hash.rs

```diff
@@ -0,0 +1,79 @@
+use pallas::crypto::hash::{Hash, Hasher};
+use pallas::ledger::primitives::alonzo;
+use pallas::ledger::primitives::alonzo::PlutusScript;
+use serde::Deserialize;
+
+use crate::{crosscut, model};
+
+#[derive(Deserialize)]
+pub struct Config {
+    pub key_prefix: Option<String>,
+    pub filter: Option<Vec<String>>,
+}
+
+pub struct Reducer {
+    config: Config,
+    address_hrp: String,
+}
+
+impl Reducer {
+    fn send_set_add(
+        &mut self,
+        slot: u64,
+        hash: Hash<28>,
+        cbor: PlutusScript,
+        output: &mut super::OutputPort,
+    ) -> Result<(), gasket::error::Error> {
+        let key = format!("{}", hash);
+        let member = hex::encode::<Vec<u8>>(cbor.into());
+
+        let crdt = model::CRDTCommand::LastWriteWins(key, member, slot);
+
+        output.send(gasket::messaging::Message::from(crdt))?;
+
+        Ok(())
```

Comment:
You mentioned that you have issue with redis, this could be partially a problem. Imagine output.send actually returns result then... if could be of type Result(), gasket::error:Error> so in some cases if sending to redis fails, it could be visible if of course sending is synchronous. For async sending this is not possible.